src/components/dropdownSearch/DropdownSearchInfiniteScrollOptions.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as InfiniteScroll from 'react-infinite-scroll-component';
+import * as _ from 'underscore';
+
+export interface DropdownSearchInfiniteScrollOptionsProps {
+    onMouseEnter: () => void;
+    options: JSX.Element[];
+    ulElementRefFunction: (menu: HTMLElement) => void;
+    infiniteScroll: InfiniteScroll.InfiniteScrollProps;
+}
+
+export class DropdownSearchInfiniteScrollOptions extends React.Component<DropdownSearchInfiniteScrollOptionsProps, {}> {
+    private id: string;
+
+    componentWillMount() {
+        this.id = _.uniqueId('infinite-dropdown');
+    }
+
+    render() {
+        return (
+            <div
+                id={this.id}
+                className='dropdown-menu'
+                ref={(menu: HTMLElement) => this.props.ulElementRefFunction(menu)}
+                onMouseEnter={() => this.props.onMouseEnter()}
+            >
+                <InfiniteScroll {...this.props.infiniteScroll} scrollableTarget={this.id} style={{...this.props.infiniteScroll.style, overflow: 'initial'}}>

```

Comment:
could you put each prop on its own line?

Reply:
I thought it was only if the line is too long. I feel like it's easier to read this way. Just like HTML.

Reply:
ok I thought the line was pretty long but maybe not long enough 👍